src/app.js

```diff
@@ -104,7 +104,7 @@ app.get(oauth2CallbackPath, cookieSession, async (req, res) => {
   }
 });
 
-if (process.env.NODE_ENV === 'production') {
+if (process.env.NODE_ENV === 'production' || process.env.NODE_ENV === 'dss') {
```

Comment:
So what are these prod and dss path's doing exactly. Can we expect ssl disabled on prod?